makefiles/pseudomodules.inc.mk

```diff
@@ -41,6 +41,7 @@ PSEUDOMODULES += gnrc_ipv6_nib_6ln
 PSEUDOMODULES += gnrc_ipv6_nib_6lr
 PSEUDOMODULES += gnrc_ipv6_nib_dns
 PSEUDOMODULES += gnrc_ipv6_nib_router
+PSEUDOMODULES += gnrc_ipv6_nib_subnets
```

Comment:
I'd prefer this its own "routing protocol" rather than a sub-module of the NIB. Making it a sub-module makes it look like it is an "official" feature of IPv6, while this more of a flashy hack than anything else. It should be doable with an "route changed" event.

Reply:
What do you mean by "route changed" event? Is there already a mechanism for this in place?

Reply:
Didn't you provide a mechanism with that with the `msgbus`? Alternatively, there is also [`gnrc_netif_ipv6_t::route_info_cb`](https://doc.riot-os.org/structgnrc__netif__ipv6__t.html#aeb73dca70bd6714fd4b6592697a32ac0) specifically for routing protocols which could be extended for prefix changes.

Reply:
msgbus would require a dedicated thread for this - and it looks like `route_info_cb` is called for each frame - that doesn't seem right.

Reply:
> msgbus would require a dedicated thread for this

What speaks against that? Would increase the modularity of your rudimentary routing protocol after all.

Reply:
> and it looks like `route_info_cb` is called for each frame - that doesn't seem right.

Only for route requests. Neighbor state changes e.g. are only called, well when the neighbor state changes. So you would just create a new hook using that function.